features/.nav

```diff
@@ -44,22 +44,21 @@
   - color.feature
   - fail_fast.feature
   - custom_settings.feature
-  - alias_example_to.feature
   - default_path.feature
 - expectation_framework_integration:
-  - configure_expectation_framework.feature
   - failure_aggregation.feature
-- mock_framework_integration:
-  - use_rspec.feature
-  - use_flexmock.feature
-  - use_mocha.feature
-  - use_rr.feature
-  - use_any_framework.feature
 - formatters:
   - text_formatter.feature
   - configurable_colors.feature
   - json_formatter.feature
   - custom_formatter.feature
-- spec_files:
-  - arbitrary_file_suffix.feature
 - core_standalone.feature
+- legacy_support_and_special_project_needs
+  - arbitrary_file_suffix.feature
+  - alias_example_to.feature
+  - use_rspec_mocks.feature
+  - use_flexmock.feature
+  - use_mocha.feature
+  - use_rr.feature
+  - use_any_framework.feature
```

Comment:
Also, it's probably worth updating the `Feature` line of the feature files to match the file name (or vice versa).  Regardless, keeping them in sync would be nice.